extensions/amp-access/amp-access.md

```diff
@@ -37,6 +40,25 @@ limitations under the License.
   </tr>
 </table>
 
+## Relationship to `amp-subscriptions`
+
+The `amp-subscriptions`extension offers similar features to `amp-access`. However
+it supports a more specilized access paywall protocol. Some notable notable
+differences are:
+
+1. The `amp-subscriptions` entitlements response is similar to the amp-access
+authorization, but it's striclty defined and standardized.
+2. The `amp-subscriptions` extension allows multiple services to be configured
+for the page to participate in access/paywall decisions. They are executed in
+parallel and paralized based on which service returns the positive response.
```

Comment:
Maybe concurrently instead of in parallel? I don't understand the "paralized".

Reply:
it think it should be prioritized (it was a carry over from the `amp-subscriptions` doc)